DistributedProofreaders/guiguts-py
# Ignore ctrl-i, ctrl-t on macOS

Testing notes for Mac:

- Press Ctrl-i, nothing should happen
- Place the cursor between two characters and press Ctrl-t, nothing should happen

I don't know the normal behavior of these keybinds on Mac/Linux, but the keybinds are behind `is_mac()` so there should be no changes on Win/Linux.

Fixes #535
